Sources/Datadog/Logger+Internal.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Logger+Internal.swift
+//  Datadog
+//
+//  Created by Maxime Epain on 24/11/2022.
+//  Copyright © 2022 Datadog. All rights reserved.
```

Comment:
Should have the usual Datadog copyright notice.

Reply:
This file shouldn't been commited.